data/events/2018/conference.yaml

```diff
@@ -0,0 +1,132 @@
+  # "We're proud to announce the following presentations for EMPEX!"
+presentations:
+  - title: "Keynote"
+    id: brooklyn-zelenka
+    presenters:
+      - zelenka-brooklyn
+    keynote: true
+    description: |
+
+  # - title: "Keynote"
+  #   id: cheng-eugenia
+  #   presenters:
+  #     - cheng-eugenia
+  #   keynote: true
+  #   description: |
+
+  # - title: Monitoring Production Elixir Applications
+  #   id: kelly-john
+  #   presenters:
+  #     - kelly-john
+  #   keynote: false
+  #   description: |
+  #     You just celebrated the launch of your first production elixir application and now have real users using your application. In this talk, you'll learn how to measure and monitor your elixir application as your user base grows from hundreds of daily users to millions. With more users comes more revenue, but also more technical challenges as you go from one server to many. And 'tail –f' is no longer useful when the logs are going by faster than you can read or the requests are load balanced across many servers. In this talk you'll learn the best practices for monitoring your applications in a way that will scale as you grow. Best of all these practices are universal, so they will be applicable if your organization is solely in the BEAM world or if you're are in a polyglot environment. Lastly, you'll learn about to measure availability in the context of web applications and how many nines of availability your API has.
+
+  # - title: Solid Ground
+  #   id: juric-sasa
+  #   presenters:
+  #     - juric-sasa
+  #   keynote: false
+  #   description: |
+  #     The BEAM, the official Erlang VM implementation, is the most important piece of the puzzle in the Elixir ecosystem. It provides some strong guarantees which minimize or completely remove some classes of problems and simplify implementation of fault-tolerant systems. It is the foundation which makes all other things in the ecosystem possible. The OTP framework, as well as third party libraries and frameworks, such as Phoenix, Ecto, or Nerves, make most sense when running on top of BEAM. Replace BEAM with something else, and you'll lose some important guarantees, which in turn can make your life much more complicated. In this talk we’ll take a closer look at a couple of seemingly simple, yet very powerful properties of BEAM, focusing in particular on some aspects of its lightweight concurrency. Going further, we’ll explore how OTP builds on top of these guarantees to bring us reliable higher-level abstractions such as supervisors. For good measure, there will be a couple of demos to make these claims more concrete.
+
+sponsor_intro: |
+sponsorship_levels:
+  - name: platinum
+    sponsor_ids:
+      - stride
+
+  - name: gold
+    sponsor_ids:
+      - sparkpost
+
+  - name: silver
+    sponsor_ids:
+      - carbon5
+
+courses:
+  - id: 1-intermed-elixir-2017
+  - id: 2-begin-elixir-2017
```

Comment:
Also planning on having a 3rd course on Nerves.  Did you want to add that now or later once details are nailed down?

Reply:
This is last year's schedule. It's currently commented out, but this will all get updated when we're ready to announce the training.